etc/singularity.conf

```diff
@@ -141,3 +163,14 @@ container dir = /var/singularity/mnt
 # better option.
 #sessiondir prefix = /var/singularity/sessions/
 
+
+# SINGULARITY USER: [STRING]
+# DEFAULT: singularity
+# The protected image mode will restrict Singularity images to those owned by
+# the 'singularity user'; this option allows admins to change the username we
+# use as a test.
+#
+# If user namespaces are enabled, the singularity user setting is ignored and
```

Comment:
I don't understand this. Is singularity the user that owns the singularity process? What kind of tests would be done and why would admins want to change this? Why does it have to be just one user?